ui/webpack.common.js

```
@@ -91,16 +90,11 @@ module.exports = (env, argv) => {
     plugins: [
       new WebpackShellPluginNext({
         onBuildStart:{
-          scripts: [
-            'echo Deleting client-gen/dist...',
-            'rm -rf client-gen/dist',
-            'echo Unzipping client-gen-' + clientGenVersion + '.zip...',
-            'unzip -q resources/client-gen-' + clientGenVersion + '.zip -d client-gen',
-            'echo Done!'
-          ],
+          scripts: [ 'resources/download-client-gen-dist.sh' ],
           blocking: true,
           parallel: false
-        }
+        },
+        safe: true,
```

this prevents silent failures of the script.